packages/vscode-extension/package.nls.json

```diff
@@ -197,6 +221,7 @@
     "teamstoolkit.handlers.installAdaptiveCardExt": "To preview and debug Adaptive Cards, we recommend to use the \"Adaptive Card Previewer\" extension.",
     "_teamstoolkit.handlers.installAdaptiveCardExt": "product name, no need to translate 'Adaptive Card Previewer'.",
     "teamstoolkit.handlers.autoInstallDependency": "Dependency installation in progress...",
+    "teamstoolkit.officeAddIn.terminal.open": "Run tasks in this terminal for office add in",
```

Comment:
Proposed: Run tasks in this terminal for Office add-in.
Logic: Office should be uppercase and needs hyphen in add-in.

Reply:
fixed.